Czaple/Models/FirstLaunch.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  FirstLaunch.swift
+//  Czaple
+//
+//  Created by Filip Trzaska on 05/12/2019.
+//  Copyright © 2019 Filip Trzaska. All rights reserved.
+//
+
+import UIKit
+
+class FirstLaunch {
+    static let instance = FirstLaunch()
+    
+    func checkFirstLaunch(window: UIWindow){
+        if UserDefaults.standard.object(forKey: "firstTime") == nil {
+            UserDefaults.standard.set(true, forKey: "firstTime")
+        }
+        UserDefaults.standard.synchronize()
```

Comment:
This is not needed anymore. Its deprecated and can be removed. It is synchronized automatically.